test/index.js

```diff
@@ -131,17 +128,25 @@ tape('should handle \'NotBeforeError\'', function (t) {
 })
 
 tape('should handle auth token', function (t) {
-  var opts = {method: 'GET', headers: {
-    Authorization: 'Bearer ' + token
-  }}
+  var now = ~~(new Date().getTime() / 1000)
+  var payload = { email: 'chet@scalehaus.io' }
+  var token = jwt.sign(payload, privateKey, { algorithm: 'RS256', expiresIn: '30d' })
+  var opts = {
+    method: 'GET',
+    headers: {
+      Authorization: 'Bearer ' + token
+    }
+  }
 
   servertest(createService(auth), '/', opts, function (err, res) {
     t.ifError(err, 'should not error')
     var data = JSON.parse(res.body)
 
     t.equal(data.email, 'chet@scalehaus.io', 'should have correct email')
-    t.equal(data.expiresIn, '30d', 'should have correct expiresIn')
-    t.ok(data.iat, 'chet@scalehaus.io', 'should have iat')
+    var expectedExpiry = now + 2592000
+    t.equal(data.exp, expectedExpiry, 'should have correct expiresIn')
```

Comment:
Since the expiry date has beed added trough sign opts, then `jsonwebtoken` package adds this as a standard jwt `exp` claim, where before it was only added as payload (not any different than email claim for example)